interview-questions.md

```diff
@@ -9,59 +9,61 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: `this` is the command/word/invocation that is ues to call an object in JavaScript.
 
+  Researched answer: The keyword `this` refers to the object that performs the current code section. It refers to the object that is performing the current function. If the reference function is a regular function, "this" refers to the global object. If the fuction is a method inside of an object, `this` refers to the object itself.
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: React is a framework (created by facebook) that allows websites to run and rerun code in an effeiacnt manner which does not required the user to constantly refresh the webpage they are viewing in order to see new content.  It is used because on its scalablity and efficiency.
```

Comment:
Awesome!  As developers, we also love react because we get immediate results too, and does a lot of the heavy lifting of application development for us.